### DISCUSSION TITLE
Personal Drive

### DISCUSSION THREAD

**Discussion by giscus** [Emilien-Foissotte/emilien-foissotte.github.io#4] [Articles]
# Personal Drive

TL;DR This blog post will cover how to setup a monitored 📊 & secure personal cloud storage ⏏️, behind a proxy ⚙️, to let you expose other nice services on your home server 🏠
Runaway from Google Drive and Google Photos 📷, host your cherry picked bytes at home 🚀 !
Intro My girlfriend recently ran into a stressfull issue, let me explain briefly the root of the problem 😬

https://emilien-foissotte.github.io/posts/2023/07/personal-drive/

**Comment by Emilien-Foissotte** [comment]
Little Bonus : 

You would like your photos to be sorted from EXIF metadata instead of modification date (the date when you copied all your files into Nextcloud) ? 

jhead is your savior on this, apply this command usint `www-data`user into your Photo directory : 

```sh
find . -iname '*.jp*g' -exec jhead -ft {} +
```

For more informations : 
- [Forum](https://www.reddit.com/r/NextCloud/comments/x3vfox/photos_dates_not_correct/)
- [StackEchange](https://photo.stackexchange.com/questions/27245/is-there-a-free-program-to-batch-change-photo-files-date-to-match-exif)

**Reply by Emilien-Foissotte** [reply]
Bonus of bonus : 

For videos files, it is similar but using `exiftool`.

Scan somes video files and display associated metadatas, using `exiftool filename.mp4`, look after a tag wich represents the creation timestamp, in my case it was either `Track Create Date`or 'Create Date` depending on the file. 

Once tag found, execute a find and a modification on theses files, in the folder where your videos are located : 

```sh
find . -iname '*.mp4' -exec exiftool '-CreateDate>FileModifyDate' {} +
```

Et voilà ! 🚀
